src/io/conteepread.c

```diff
@@ -0,0 +1,104 @@
+#include "macros.h"
+#include "PR/rcp.h"
+#include "controller.h"
+#include "siint.h"
+
+static void __osPackEepReadData(u8 address);
+OSPifRam __osEepPifRam ALIGNED(16);
+s32 __osEepromRead16K;
+s32 osEepromRead(OSMesgQueue *mq, u8 address, u8 *buffer)
+{
+	s32 ret;
+	int i;
+	u16 type;
+	u8 *ptr;
+	OSContStatus sdata;
+	__OSContEepromFormat eepromformat;
+
+	ret = 0;
+	i = 0;
```

Comment:
```suggestion
	s32 ret = 0;
	s32 i = 0;
	u16 type;
	u8 *ptr;
	OSContStatus sdata;
	__OSContEepromFormat eepromformat;

```
Disregard the type change if it's from mdebug

Reply:
It's int in the mdebug.